adobe/spectrum-web-components
# fix(menu): prevent sp-menu-item text-align cascading

## Description

Previous to the Overlay V2, the `text-align` property of action-menu, or similar components, did not cascade into the `sp-menu-items`. This PR brings back that behavior by setting the `text-align: initial` on `sp-menu-item`. 

PS: This is the same behavior as in React Spectrum. As per Spectrum guidelines I haven't yet seen a use-case for other types of aligment. If that's the case we can provide a `--mod-*` type css variable.

## Related issue(s)

- TBD

## Motivation and context

Fixes the text-align cascading into the overlay menu-items.

## How has this been tested?




    1. Open ActionMenu Storybook
    2. Place a `text-align: center` on a parent of the ActionMenu (e.g. sp-theme) 
    3. Click the ActionMenu
    4. Observe that the text inside the menu-items is centered


## Screenshots (if appropriate)
Without the change:

After the change:
<img width="1275" alt="storybook current" src="https://github.com/user-attachments/assets/bfa9ba95-5d39-4c1b-9fe0-acf5cd39d6e5">



## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
